clients/src/main/resources/common/message/ProduceResponse.json

```diff
@@ -44,7 +47,11 @@
         { "name": "LogAppendTimeMs", "type": "int64", "versions": "2+", "default": "-1", "ignorable": true,
           "about": "The timestamp returned by broker after appending the messages. If CreateTime is used for the topic, the timestamp will be -1.  If LogAppendTime is used for the topic, the timestamp will be the broker local time when the messages are appended." },
         { "name": "LogStartOffset", "type": "int64", "versions": "5+", "default": "-1", "ignorable": true,
-          "about": "The log start offset." }
+          "about": "The log start offset." },
+        { "name":  "ErrorRecords", "type":  "[]int32", "versions": "8+", "default": "", "ignorable":  true,
+          "about":  "The relative offset of records that cause the batch to be dropped"},
```

Comment:
Here, we should have the same const string of `The relative offset of records that cause the batch to be dropped` in the java code as well for consistency.

Reply:
ack